setup.py

```diff
@@ -44,7 +44,7 @@ def get_docs_url():
         "python-can==4.3.1",
         "ingenialogger>=0.2.1",
         "ping3==4.0.3",
-        "pysoem>=1.1.6, <1.2.0",
+        "pysoem@ git+https://github.com/bnjmnp/pysoem.git#master",
```

Comment:
Isn't this released yet? If we install pysoem like this, SOEM would not be compiled, right?

Reply:
You're right, we are missing the build tools in the Jenkins agents.